trezor-common/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "trezor-common"
+license.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+num-derive.workspace = true
+num-traits = { version = "0.2", default-features = false, features = ["libm"] }
+parity-scale-codec = { version = "3.1", default-features = false, features = ["derive", "chain-error"] }
+
+[dev-dependencies]
+test-utils = { path = "../test-utils" }
+common = { path = "../common" }
+crypto = { path = "../crypto/" }
+
+bitcoin-bech32.workspace = true
```

Comment:
Looks like only rstest dependency is actually used

Reply:
removed